src/main/java/emissary/kff/KffFile.java

```diff
@@ -31,16 +34,17 @@ public class KffFile implements KffFilter {
     private final Logger logger;
 
     /** File containing SHA-1/CRC32 results of known files */
-    protected RandomAccessFile knownFile;
+    // protected RandomAccessFile knownFile;
```

Comment:
Jumping in here late so sorry if I am missing some context.
Any reason to just remove the line instead of commenting out `RandomAccessFile`
Same for line 89 as well.

Not blocking approval. There are a few other smells here that we can clean up in a later pr if we want